packages/core/src/components/breadcrumb/breadcrumb.css

```diff
@@ -0,0 +1,71 @@
+.hds-breadcrumb {
+  display: flex;
+  padding: var(--spacing-s) 0;
+}
+
+.hds-breadcrumb__list,
+.hds-breadcrumb__list {
+  flex-direction: row;
+  align-items: center;
+  word-wrap: break-word;
+}
+
+.hds-breadcrumb__list--mobile {
+  display: flex;
+}
+
+.hds-breadcrumb__list--desktop {
+  display: none;
+}
+
+.hds-breadcrumb__list-item {
+  align-items: center;
+  display: flex;
+  line-height: var(--lineheight-l);
+}
+.hds-breadcrumb__list-item--active {
+  font-weight: 700;
+  /* to match 3px border around links */
+  padding: 3px;
+}
+
+.hds-breadcrumb__link.hds-breadcrumb__link {
+  --link-visited-color: none;
+  --link-color: var(--color-black-90);
+}
+
+.hds-breadcrumb__link:hover,
```

Comment:
Hover style is missing. The underline should be gone on hover.

Reply:
How do you know this? 😄 It's now in the design. It is on the hel.fi -site, but I don't know if we use those styles.

Reply:
😂 Yeah I noticed it on hel.fi but didn't see it on the designs but I think there should be something for hover. I think it would be good to confirm this with @hannu-kivimaki

Reply:
Currently there is no hover effect, but it is using HDS Link, which has no hover effect either.

Reply:
Ah, but why not use Header.NavigationLink? 😉 It could be better to use that as it's meant for navigational links. Link component is meant for links in text. Although the NavigationLink component will receive some (maybe major) refactoring when all the PRs are merged to the same branch. It needs to get rid of Children API (used with dropdowns) and to support `as` prop.

Reply:
NavigationLink has different styles (font-size, underline, padding) and it is not in core. Not much gained by using it.

Reply:
Designer says underline stays.

Reply:
So be it. The tribe has spoken. 🔥